packages/demo/src/app/bootstrap/components/alert/action-alert-demo/action-alert-demo.component.scss

```diff
@@ -1,6 +1,6 @@
 // Import variables
-@use '@swisspost/design-system-styles/cwf';
+@use '@swisspost/design-system-styles/core' as post;
 
 .pi-2063-white {
-  @include cwf.pi(2063, 'white');
+  @include post.pi(2063, 'white');
```

Comment:
This could create a task on the `Styles Next` board:
Rename mixin `pi` for example to `icon`, so it can be executed as `post.icon(2063, 'white')`.

Reply:
Good idea, can you create a ticket?

Reply:
Here we go #392